backend/sources/facebook/connector/src/main/java/co/airy/core/sources/facebook/api/Api.java

```diff
@@ -51,16 +57,15 @@
     private final HttpHeaders httpHeaders = new HttpHeaders();
     private final String appId;
     private final String apiSecret;
-    private static final String errorMessageTemplate =
-            "Exception while sending a message to Facebook: \n" +
-                    "Http Status Code: %s \n" +
-                    "Error Message: %s \n";
 
-    public Api(ObjectMapper objectMapper, RestTemplateBuilder restTemplateBuilder,
```

Comment:
Here we use inject the default airy object mapper, but it's better to explicitly define one for Facebook here.